node/src/types/exit_code.rs

```diff
@@ -18,9 +18,6 @@ pub enum ExitCode {
     /// The process should exit with `101`, equivalent to panicking.  The launcher should not
     /// restart the node.
     Abort = 101,
-    /// The process should exit with `102`.  The launcher should proceed to run the previous
-    /// installed version of `casper-node`.
-    DowngradeVersion = 102,
```

Comment:
The exit codes are used to integrate node with the launcher. If we remove this exit code, will the launcher be updated as well to not handle it? Maybe we should leave a comment so that we don't use `102` as an exit code in the future?

Reply:
Addressed in: cc389c66a0864b8b3a89f4c2dc1648dc241cb7fc

The fact that this enum is `pub` makes it unnecessary for the `#allow(unused)` flag to be added. I marked it with `#[doc(hidden)]` though.

Reply:
Awesome, thanks.